docs/runbooks/KubeVirtDeprecatedAPIsRequested.md

```diff
@@ -0,0 +1,36 @@
+# KubeVirtDeprecatedAPIsRequested
+<!-- Edited by fmatouschek, May 2023-->
+
+## Meaning
+
+This alert fires when a deprecated KubeVirt API was requested.
```

Comment:
"was requested" > "is requested"
Our style guidelines specify using the present tense unless the context is obviously future ("they will be removed in a future release").